commands/utils/SourceArchiveUtility.ts

```diff
@@ -73,42 +72,42 @@ export async function scheduleRunRequest(fileUri: vscode.Uri, requestType: runRe
     }
 
     //Schedule the run and Clean up.
-    status1.appendLine("Set up Run Request");
+    ext.outputChannel.appendLine("Set up Run Request");
 
     const run = await client.registries.scheduleRun(resourceGroupName, registry.name, runRequest);
-    status1.appendLine("Scheduled Run " + run.runId);
+    ext.outputChannel.appendLine("Scheduled Run " + run.runId);
 
-    await streamLogs(registry, run, status1, client);
+    await streamLogs(registry, run, client);
     await fse.unlink(tarFilePath);
 }
 
-async function uploadSourceCode(status: vscode.OutputChannel, client: ContainerRegistryManagementClient, registryName: string, resourceGroupName: string, rootFolder: vscode.WorkspaceFolder, tarFilePath: string): Promise<string> {
-    status.appendLine("   Sending source code to temp file");
+async function uploadSourceCode(client: ContainerRegistryManagementClient, registryName: string, resourceGroupName: string, rootFolder: vscode.WorkspaceFolder, tarFilePath: string): Promise<string> {
+    ext.outputChannel.appendLine("   Sending source code to temp file");
     let source: string = rootFolder.uri.fsPath;
     let items = await fse.readdir(source);
     items = items.filter(i => !(i in vcsIgnoreList));
     // tslint:disable-next-line:no-unsafe-any
     tar.c({ cwd: source }, items).pipe(fse.createWriteStream(tarFilePath));
 
-    status.appendLine("   Getting Build Source Upload Url ");
+    ext.outputChannel.appendLine("   Getting Build Source Upload Url ");
     let sourceUploadLocation = await client.registries.getBuildSourceUploadUrl(resourceGroupName, registryName);
     let upload_url: string = sourceUploadLocation.uploadUrl;
     let relative_path: string = sourceUploadLocation.relativePath;
 
-    status.appendLine("   Getting blob info from Upload Url ");
+    ext.outputChannel.appendLine("   Getting blob info from Upload Url ");
     // Right now, accountName and endpointSuffix are unused, but will be used for streaming logs later.
     let blobInfo = getBlobInfo(upload_url);
-    status.appendLine("   Creating Blob Service ");
+    ext.outputChannel.appendLine("   Creating Blob Service ");
     let blob: BlobService = createBlobServiceWithSas(blobInfo.host, blobInfo.sasToken);
-    status.appendLine("   Creating Block Blob ");
+    ext.outputChannel.appendLine("   Creating Block Blob ");
     blob.createBlockBlobFromLocalFile(blobInfo.containerName, blobInfo.blobName, tarFilePath, (): void => { });
     return relative_path;
 }
 
-function getTempSourceArchivePath(status: vscode.OutputChannel): string {
+function getTempSourceArchivePath(): string {
     /* tslint:disable-next-line:insecure-random */
     let id: number = Math.floor(Math.random() * Math.pow(10, idPrecision));
-    status.appendLine("Setting up temp file with 'sourceArchive" + id + ".tar.gz' ");
+    ext.outputChannel.appendLine("Setting up temp file with 'sourceArchive" + id + ".tar.gz' ");
```

Comment:
It would be better to create a variable with the filename first and use it in both places.